src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -59,16 +61,14 @@
 	private JMenu fileMenu;
 	private JMenuItem saveMenuItem;
 
-	private JMenu loadMenu;
-	private JMenuItem loadMandelbrotMenuItem;
-	private JMenuItem loadJuliaMenuItem;
-
 	private JMenuItem saveImageMenuItem;
+	private JMenuItem loadMenuItem;
+	private JMenuItem helpMenuItem;
 	private JMenuItem exitMenuItem;
 
 	private JMenu optionsMenu;
 	private JMenuItem createColorPaletteMenuItem;
-	private JMenuItem setJuliaConstantsMenuItem;
+	private JMenuItem setConstantsMenuItem;
```

Comment:
This variable name is confusing - should it just be constantsMenuItem? Why does it have the prefix 'set'?